README.md

```diff
@@ -220,6 +220,64 @@ C:\Users\bion\Desktop>dumpbin /IMPORTS .\example.exe | findstr msvcp
+-DENABLE_TESTS=TRUE -S . -B {wherever you want binaries}`. This differs from above only in `-DENABLE_TESTS=TRUE`.
+5. If you have already followed the steps from
+[How To Build With A Native Tools Command Prompt](#how-to-build-with-a-native-tools-command-prompt), and have not
+changed the value of {wherever you want binaries} in step four, then there is no need to rebuild to run the tests.
+Otherwise, invoke `ninja -C {wherever you want binaries}` to build the project.
+
+In addition to following the above steps you must also have [Python][] 3.8 or newer, and have LLVM's `bin` directory on
+the PATH.
+
+## Running All The Tests
+
+After configuring and building the project, running `ctest` from the build output directory will run all the tests.
+CTest will only display the standard error output of tests that failed. In order to get more details from CTest's
+`lit` invocations, run the tests with `ctest -V`.
+
+## Running A Subset Of The Tests
+
+`${PROJECT_BINARY_DIR}\tests\llvm-lit\llvm-lit.py` can be invoked on a subdirectory of a testsuite and will execute all
+the tests under that subdirectory. This can mean executing the entirety of a single testsuite, running all tests under
+a category in libcxx, or running a single test in `std` and `tr1`.
+
+## Examples
+
+```
+:: This command configures the project with tests enabled. It assumes you are using the vcpkg submodule and have
+:: already installed boost. It also assumes you have inited and updated the llvm-project submodule.
+C:\STL\build>cmake -GNinja -DCMAKE_CXX_COMPILER=cl -DCMAKE_TOOLCHAIN_FILE=..\vcpkg\scripts\buildsystems\vcpkg.cmake -DENABLE_TESTS=TRUE ..
+
+:: As stated above this step is only strictly necessary if you have yet to build the STL or if you have changed the
+:: output directory of the binaries. Any changes or additions in any of the existing testsuites do not require
+:: recompilation or reconfiguration to take effect when running the tests.
+C:\STL\build>ninja
+
+:: This command will run all of the testsuites with verbose output.
+C:\STL\build>ctest -V
+
+:: This command will run all of the tr1 testuite
+C:\STL\build>ctest -R tr1
+
+:: This command will also run all of the tr1 testuite.
+C:\STL\build>python tests\llvm-lit\llvm-lit.py ..\tests\tr1
+
+:: If you want to run a subset of a testsuite you need to point it to the right place in the sources. The following
+:: will run the single test found under VSO_0000000_any_calling_conventions.
+C:\STL\build>python tests\llvm-lit\llvm-lit.py ..\tests\std\tests\VSO_0000000_any_calling_conventions
+
+:: You can invoke llvm-lit with any arbitrary subdirectory of a testuite. In libcxx this allows you to have a finer
+:: control over what category of tests you would like to run. The following will run all the libcxx map tests.
+C:\STL\build>python tests\llvm-lit\llvm-lit.py ..\llvm-project\libcxx\test\std\containers\associative\map
+```
+

```

Comment:
Should we have another section describing how to interpret the results?

Reply:
Sure I guess that's worth it since `ctest` has different output than `llvm-lit`.

Reply:
...and our user may be unfamiliar with both.